### ISSUE TITLE
Jackknife CI: biased or bias-corrected?

### ISSUE THREAD

**Issue by DavorJ** [imbs-hl/ranger#753]
`predict.ranger` has an implementation ([`type = "se"`](https://www.rdocumentation.org/packages/ranger/versions/0.16.0/topics/predict.ranger)) of [Wager (2014)](https://jmlr.org/papers/v15/wager14a.html) for the calculation of confidence intervals for RF predictors. 

What is unclear from the documentation is whether this is the biased or bias-corrected estimate. Maybe explicitely state this? From the code (#188) I gather they are bias-corrected estimates.

**Comment by mnwright** [comment]
Yes, it's the bias-corrected estimate and we should write that in the docs (sorry for the late reply).
